docs/source/quickstarts/deploy-a-large-language-model-with-openllm-and-bentoml.rst

```diff
@@ -85,30 +122,51 @@ The server is now active at `http://0.0.0.0:3000 <http://0.0.0.0:3000/>`_. You c
 
     .. tab-item:: CURL
 
+        For one-shot generation
+
         .. code-block:: bash
 
-         curl -X 'POST' \
-            'http://0.0.0.0:3000/prompt' \
-            -H 'accept: text/plain' \
-            -H 'Content-Type: text/plain' \
-            -d '$PROMPT' # Replace $PROMPT here with your prompt.
+
+           curl -X 'POST' \
+               'http://0.0.0.0:3000/v1/generate' \
```

Comment:
Shouldn't "sampling_params" be included in the curl request? I omitted it and I got this:

```
Traceback (most recent call last):
  File "/home/paperspace/.local/lib/python3.9/site-packages/bentoml/_internal/server/http_app.py", line 341, in api_func
    output = await api.func(*args)
  File "/home/paperspace/Documents/service.py", line 28, in generate
    n = request['sampling_params'].pop('n', 1)
KeyError: 'sampling_params'
```

Reply:
fixed.